go.mod

```diff
@@ -67,3 +67,5 @@ require (
 	google.golang.org/protobuf v1.28.0 // indirect
 	gopkg.in/yaml.v3 v3.0.1 // indirect
 )
+
+replace github.com/newrelic/newrelic-client-go/v2 => /Users/nzur/Documents/GitHub/NewServices/newrelic-client-go
```

Comment:
This should not be here

Reply:
It's only relevent for the development while I'm waiting for the end of the freeze.

Reply:
removed